lib/waffle/storage/local.ex

```diff
@@ -42,20 +42,24 @@ defmodule Waffle.Storage.Local do
   end
 
   def url(definition, version, file_and_scope, _options \\ []) do
-    local_path = Path.join([
-      definition.storage_dir(version, file_and_scope),
-      Versioning.resolve_file_name(definition, version, file_and_scope)
-    ])
+    path = path(definition, version, file_and_scope)
     host = host(definition)
 
     if host == nil do
-      Path.join("/", local_path)
+      Path.join("/", path)
     else
-      Path.join([host, local_path])
+      Path.join(host, path)
     end
     |> URI.encode()
   end
 
+  def path(definition, version, file_and_scope) do
```

Comment:
if it returns path in storage, we probably should include prefix path for local storage

Reply:
That's a good point. I have updated `Local.path/3` to behave more like `url/3` while never adding the asset host. That should produce a usable path.